WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -561,9 +561,6 @@ function WeakAuras.regionPrototype.AddSetDurationInfo(region)
     -- In that case we also need to overwrite TimerTick
     region.SetDurationInfo = function(self, duration, expirationTime, customValue, inverse)
       self.duration = self.duration or 0
-      if duration <= 0 or duration > self.duration or true then
-        self.duration = duration;
```

Comment:
shouldn't you keep `self.duration = duration` ?

Reply:
Good call, but then we can just skip the first part entirely